src/report/issue_closure.rs

```diff
@@ -1,12 +1,14 @@
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
 use fehler::throws;
+use log::debug;
 use serde::{Deserialize, Serialize};
 use stable_eyre::eyre::{Error, WrapErr};
 
 use super::{Report, ReportConfig, ReportData};
 
-use std::fs::File;
-use std::path::Path;
-
 #[derive(Debug, Serialize, Deserialize)]
 pub(super) struct IssueClosure {
```

Comment:
We don't need this struct at all any more. We just need the `write_issue_closures` function which can read info from `ReportData.repo_infos`